pkg/pipeline/sink/segments.go

```diff
@@ -55,6 +55,10 @@ func newSegmentSink(u uploader.Uploader, conf *config.PipelineConfig, p *config.
 	if maxDuration == 0 {
 		maxDuration = time.Hour * 48
 	}
+	segmentDuration := p.SegmentDuration
```

Comment:
handled in pkg/config/output, should default to either KeyFrameInterval or 4

Reply:
was there actually a panic here? That would mean something is broken in the config